File: DailyAdventureChallenge/app/src/main/java/com/example/dailyadventurechallenge/data/dto/Challenge/ChallengeResponseDTO.kt
package com.example.dailyadventurechallenge.data.dto.Challenge

import com.google.gson.annotations.SerializedName

data class ChallengeResponseDTO(
    @SerializedName("idChallenge")
    val idChallenge: Int,

    @SerializedName("description")
    val description: String?,

    @SerializedName("difficulty")
    val difficulty: String?,

    @SerializedName("points")
    val points: Int
)

File: DAC_API/Models/DTO/Challenge/ChallengeResponseDTO.cs
namespace DAC_API.Models.DTO.Challenge {
    public class ChallengeResponseDTO {
        public int IdChallenge { get; set; }
        public string? Description { get; set; }
        public string? Difficulty { get; set; }
        public int? Points { get; set; }
    }
}
